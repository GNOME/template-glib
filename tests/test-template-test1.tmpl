<html>
 <head>
  <title>{{title}}</title>
 </head>
 <body>
{{if title}}
  <h1>{{title}}</h1>
{{else}}
  <h1>Untitled</h1>
{{end}}
 </body>
</html>
