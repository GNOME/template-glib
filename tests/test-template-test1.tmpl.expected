<html>
 <head>
  <title>My Title</title>
 </head>
 <body>
  <h1>My Title</h1>
 </body>
</html>
